hardware/Schematic-Design-Guide.md

```diff
@@ -72,16 +95,22 @@ After you get the reference design, you may have the concern why the reference i
 
   - Low Frequency Clock Sources
 
-    An external low frequency clock can be supplied from a crystal/ceramic resonator or from an external clock source. It can source a low-frequency clock from an external source such as a TCXO or VCXO.
-  ![](files/HW-Schematic-Design-Guide/LF-Connection.png)
+    An external low frequency clock can be supplied from a crystal/ceramic resonator or an external clock source. It can source a low-frequency clock from an external source such as a TCXO or VCXO.
+    <div align="center">
+     <img src="files\HW-Schematic-Design-Guide\LF-Connection.png">  
+    </div>
+  
   - High Frequency Clock Sources
 
-    The high frequency clock can be sourced from a crystal or ceramic resonator or from an external square or sine wave source. It can source a high-frequency clock from an external source such as a TCXO or VCXO.
-   ![](files/HW-Schematic-Design-Guide/HF-Connection.png) 
-For additional information on the external oscillators, refer to the application note, [AN0016.1](https://www.silabs.com/documents/public/application-notes/an0016.1-efm32-series-1-oscillator-design-considerations.pdf)
+    The high frequency clock can be sourced from a crystal or ceramic resonator or an external square or sine wave source. It can source a high-frequency clock from an external source such as a TCXO or VCXO.
+    <div align="center">
+     <img src="files\HW-Schematic-Design-Guide\HF-Connection.png">  
+    </div>
+   
+For additional information on the external oscillators, refer to the application note, [AN0016.1](https://www.Silabs.com/documents/public/application-notes/an0016.1-efm32-series-1-oscillator-design-considerations.pdf)
 
-## RF circuits matching guide 
-The EFR32 Wireless Gecko Series devices include chip variants that provide 2.4 GHz-only operation, sub-GHz-only operation, or dual-band (2.4GHz and sub-GHz)operation.For RF matching circuits design of these bands, there are 2 application notes described the methods. One is for sub-GHz band [AN923](https://www.silabs.com/documents/public/application-notes/AN923-subGHz-Matching.pdf), and the other is for 2.4GHz band [AN930](https://www.silabs.com/documents/public/application-notes/an930-efr32-matching-guide.pdf).
+# 4. RF circuits matching guide 
+The EFR32 Wireless Gecko Series devices include chip variants that provide 2.4 GHz-only operation, sub-GHz-only operation, or dual-band (2.4GHz and sub-GHz)operation.For RF matching circuits design of these bands, there are 2 application notes described the methods. One is for sub-GHz band [AN923](https://www.Silabs.com/documents/public/application-notes/AN923-subGHz-Matching.pdf), and the other is for 2.4GHz band [AN930](https://www.Silabs.com/documents/public/application-notes/an930-efr32-matching-guide.pdf).
```

Comment:
With the AN930 reference, I think we should mention this is for series 1 EFR32 and AN930.2 should be referenced for series 2 EFR32

Reply:
Added.